src/docs/exampleComponents/ComposedChart/BandedChart.js

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import {
+  ComposedChart,
+  Line,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  ResponsiveContainer,
+} from 'recharts';
+
+const data = [
+    {
+      name: "Page A",
+      a: [0, 0],
+      b: 0,
+    },
+    {
+      name: "Page B",
+      a: [50, 300],
+      b: 106,
+    },
+    {
+      name: "Page C",
+      a: [150, 423],
+    },
+    {
+      name: "Page D",
+      b: 312,
+    },
+    {
+      name: "Page E",
+      a: [367, 678],
+      b: 451,
+    },
+    {
+      name: "Page F",
+      a: [305, 821],
+      b: 623,
+    },
+  ];
+  
+  export default class Example extends PureComponent {
+    renderTooltipWithoutRange = ({ payload, content, ...rest }) => {
+        const newPayload = payload.filter((x) => x.dataKey !== "a");
```

Comment:
I have not found a better way to exclude "band" from the tooltip. Is there anything less verbose?

Reply:
Eh I don't think so. You could try to use the formatter function? But it gives you less control

Reply:
`formatter()` controls only how value is rendered; even if formatter returns `null` or `undefined`, there still will be the row "a:" in the tooltip:

![2024-05-25 17_11_10](https://github.com/recharts/recharts.org/assets/956687/ce2a9d8c-a667-4109-90f7-33cdc4e1d72c)

Reply:
Ack yeah, I haven't used it in a while. I think like you have is the only way